CMSgov/design-system
# [WNMGDS-1373] Gatsby core doc site

## Summary

Adding the `docs` directory to house our new Gatsby-powered documentation site. This will replace the 'core' doc site.

Still to come:
* updates to doc site look & feel
* migrate child systems' content

## How to test

`yarn build-storybook:gatsby` to build storybook into docs directory
`yarn build:gatsby` & `yarn serve:gatsby` to see the doc site locally
